### PR TITLE
Introduce OncoKB Card for OncoKB integration under annotation column

### PR DESCRIPTION
**Pull request by zhx828**
# What? Why?
Improve OncooKB annotation integration in cBioPortal

# Checks
- [x] Runs on Heroku.
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [x] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

# Any screenshots or GIFs?

screen shot 2016-07-22 at 12 13 28 am

screen shot 2016-07-22 at 12 13 39 am

# Notify reviewers
@cBioPortal/frontend

### CODE REVIEW

**Comment by zhx828** [comment]
@fedde-s @pieterlukasse I may need to make 508 compliance working well in OncoKB template. Could you tell me which attributes I need to put in and on which HTML tag? Thanks.

**Comment by pieterlukasse** [comment]
@zhx828 you can check some of our PRs and tickets like #1351, and https://github.com/cBioPortal/cbioportal/pulls?q=is%3Apr+508+is%3Aclosed . In #1351 you can also find a screenshot of the WAVE tool that can be used for this. This tool will tell you what needs to be fixed and how. 

@fedde-s can probably give you a summary of most common changes he had to make.

**Comment by zhx828** [comment]
@fedde-s when you have time, could you take a look the 508 compliance commit? Thank you.

**Comment by fedde-s** [comment]
I do not have time to find out how to configure OncoKB integration in a local deployment and run the WAVE tool on it by today, but by eye I do not see any newly introduced input elements without labels or title attributes, nor images missing an alt text that conveys the information in the image, so that looks fine. The column header scope declarations may not even be strictly necessary in such a simple table where the `<th>` tags are in the first row, but being explicit doesn’t hurt 👍

One small question: do you ever expect the variables `title` in `oncokb-card-template.html` and `_datum` in the method `OncoKB.utils.findRegex()` to be empty strings? That could result in links without text.

**Comment by zhx828** [comment]
@fedde-s `OncoKB.utils.findRegex()` handles empty string. And I think it's users' decision whether they want to use empty title.

File: src/main/resources/portal.properties.EXAMPLE [hunk 1]
```diff
@@ -121,7 +121,7 @@ tcga_path_report.url=https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftp
 segfile.url=http://cbio.mskcc.org/cancergenomics/gdac-portal/seg/
 
 # Enable OncoKB annotation
-oncokb.url=
+oncokb.url=http://oncokb.org/legacy-api/
```
Comment (jjgao) [inline src/main/resources/portal.properties.EXAMPLE]: you can also add some code to GloabalProperties.java to use this url by default if oncokb.url is empty of null.

Comment (jjgao) [inline src/main/resources/portal.properties.EXAMPLE]: maybe also add a new line:
`use_oncokb=true`
Unless it is set to false, we always display oncokb.

**Pull request merged**